activiti-cloud-notifications-graphql-service/services/events/src/main/java/org/activiti/cloud/services/notifications/graphql/events/consumer/EngineEventsConsumerMessageHandler.java

```diff
@@ -44,25 +44,25 @@ public EngineEventsConsumerMessageHandler(Transformer transformer,
         this.transformer = transformer;
     }
 
-    @StreamListener
-    public void receive(@Input(EngineEventsConsumerChannels.SOURCE)
-                            Flux<Message<List<Map<String,Object>>>> input) {
+    @StreamListener(EngineEventsConsumerChannels.SOURCE)
+    public void receive(Message<List<Map<String, Object>>> input) {
 
         // Let's process and transform message from input stream
-        input.flatMapSequential(message -> {
-            List<Map<String, Object>> events = message.getPayload();
-            String routingKey = (String) message.getHeaders().get("routingKey");
+        Flux.just(input)
+                .flatMapSequential(message -> {
```

Comment:
@jesty @erdemedeiros This may be a workaround, until we can switch to native reactive binding..

Reply:
@igdianov, so are you happy that for now? Can we merge the PR and come back to it later on?

Reply:
@mteodori already created an issue about his migration (AAE-2608)